.github/workflows/build-and-push.yml

```
@@ -0,0 +1,27 @@
+name: Build to release branches
+
+on:
+    workflow_dispatch:
+    push:
+        tags: ['*']
+        branches:
+            - dev-beta
+            - dev-main
+
+concurrency:
+    group: ${{ github.workflow }}-${{ github.ref }}
+    cancel-in-progress: false
+
+jobs:
+    build-assets:
+        uses: inpsyde/reusable-workflows/.github/workflows/build-and-push-assets.yml@add_option_to_select_built_branch_name
+        with:
+            PHP_VERSION: '8.1'
```

IIRC, we said that the workflows should use the minimum required PHP version, which is 8.0, according to `composer.json`:
https://github.com/RRZE-Webteam/FAU-Studium/blob/1cccae5c4b42d94c6b3404d892e86b7154627eaf/composer.json#L15
```suggestion
            PHP_VERSION: '8.0'
```